e2e/LongHaul/prerequisites/LoadEnvironmentVariablesFromKeyVault.ps1

```diff
@@ -0,0 +1,53 @@
+param(
```

Comment:
This is the same as the file under e2e\Tests\prerequisites, right?

Let's just use the same one. We can move it up to be under e2e directly.